ui/src/app/components/gke-app-configuration-panels/create-gke-app.tsx

```diff
@@ -401,7 +411,41 @@ export const CreateGkeApp = ({
           </FlexColumn>
         </FlexRow>
       </div>
+      <div style={{ ...styles.controlSection }}>
+        <FlexRow>
+          <TooltipTrigger
+            disabled={!disableDiskSizeSelector}
+            content={'Cannot modify Disk size if App or Persistent Disk exist'}
```

Comment:
I also think we should split these 2 cases, because they're different from the user's perspective.

app exists -> user sees this as "the app is already running" so maybe "cannot modify a running application"

disk exists -> user does not see a running app and may not realize the disk is present.  So in this case it's something like "cannot modify the existing RStudio disk.  You must delete it if you wish to create one with a different size"

Reply:
Something like this:
![Screenshot 2024-05-13 at 4 03 31 PM](https://github.com/all-of-us/workbench/assets/34481816/f55d00a5-5503-484f-bbf9-4954a3a6ada2)
![Screenshot 2024-05-13 at 4 03 39 PM](https://github.com/all-of-us/workbench/assets/34481816/a0358582-0098-4812-a70a-a71d46bafffb)